bundles/org.openhab.binding.broadlink/README.md

```diff
@@ -0,0 +1,126 @@
+| pollingInterval     | Integer | 30            | The interval in seconds for polling the status of the device                      |
+| ignoreFailedUpdates | Boolean | false         | If enabled, failed status requests won't put the device `OFFLINE`                       |
+| mapFilename         | String  | broadlink.map | The map file that contains remote codes to send via IR                            |
+
+> The `mapFilename` setting is applicable to the RM series of devices only.
+
+## Channels
+
+| Channel          | Supported Devices        | Type                 | Description                                     |
+|------------------|--------------------------|----------------------|-------------------------------------------------|
+| powerOn          | MP2, all SPx             | Switch               | Power on/off for switches/strips                |
+| nightLight       | SP3                      | Switch               | Night light on/off                              |
+| temperature      | A1, RM2, RM4             | Number:Temperature   | Temperature                                     |
+| humidity         | A1, RM4                  | Number:Dimensionless | Air humidity percentage                         |
+| noise            | A1                       | String               | Noise level: `QUIET`/`NORMAL`/`NOISY`/`EXTREME` |
+| light            | A1                       | String               | Light level: `DARK`/`DIM`/`NORMAL`/`BRIGHT`     |
+| air              | A1                       | String               | Air quality: `PERFECT`/`GOOD`/`NORMAL`/`BAD`    |
+| s1powerOn        | MP1, MP1_1k3s2u          | Switch               | Socket 1 power                                  |
+| s2powerOn        | MP1, MP1_1k3s2u          | Switch               | Socket 2 power                                  |
+| s3powerOn        | MP1, v_1k3s2u            | Switch               | Socket 3 power                                  |
+| s4powerOn        | MP1                      | Switch               | Socket 4 power                                  |
+| usbPowerOn       | MP1_1k3s2u               | Switch               | USB power                                       |
+| powerConsumption | MP2, SP2s,SP3s           | Number:Power         | Power consumption                               |
+| command          | all RMx                  | String               | IR Command code to transmit                     |
+| learningControl  | all RMx                  | String               | Learn mode command channel (see below)          |
+
+## Map File
+
+The Broadlink RM family of devices can transmit IR codes.
+The map file contains a list of IR command codes to send via the device.
+The file uses the [Java Properties File format](https://en.wikipedia.org/wiki/.properties) and is stored in the `<OPENHAB_CONF>/transform` folder.
```

Comment:
I'm pretty sure the default should not be the conf/transform folder. From having other bindings get merged, I was told the correct place is this using the binding name as a sub folder.....
<OPENHAB_USER_DATA>/broadlink/<UID>.map
or
<OPENHAB_USER_DATA>/broadlink/<UID>/broadlink.map

This location will have write permissions on mac/win/linux
Also consider what happens if someone has multiple things, hence the suggestion to use the Unique ID of the thing in the filename or a sub folder, whatever makes sense.

Use this code to get the location...

OpenHAB.getUserDataFolder() + "/broadlink/" + this.thing.getUID().getId() + "/"

I like the idea of it in a file so you can share the files to others easily.